<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">Line {
	background-color: #666666;
	}
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>950</width>
      <height>517</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icon_background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="left_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
	border: none;
	background-color: blue;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="Settings_but">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="notification_but">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>145</y>
      <width>531</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF;
font-size: 16px;
font-family: Calibri, Candara, Segoe, &quot;Segoe UI&quot;, Optima, Arial, sans-serif;
font-style:normal;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Let's start</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="under_line">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_2">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="day">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;
font-size: 24px;
font-weight: 900;
font-family: Calibri, Candara, Segoe, &quot;Segoe UI&quot;, Optima, Arial, sans-serif;
font-style:normal;</string>
    </property>
    <property name="text">
     <string>Today</string>
    </property>
   </widget>
   <widget class="QLabel" name="Data">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="FirstName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;
font-size: 20px;
font-family: Calibri, Candara, Segoe, &quot;Segoe UI&quot;, Optima, Arial, sans-serif;
font-style:normal;</string>
    </property>
    <property name="text">
     <string>FirstName</string>
    </property>
   </widget>
   <widget class="QLabel" name="SecondName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;
font-size: 20px;
font-family: Calibri, Candara, Segoe, &quot;Segoe UI&quot;, Optima, Arial, sans-serif;
font-style:normal;</string>
    </property>
    <property name="text">
     <string>SecondName</string>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>141</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dial_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Avatar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>118</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>left_button</zorder>
   <zorder>Settings_but</zorder>
   <zorder>notification_but</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>Data</zorder>
   <zorder>FirstName</zorder>
   <zorder>day</zorder>
   <zorder>SecondName</zorder>
   <zorder>dial</zorder>
   <zorder>dial_2</zorder>
   <zorder>Avatar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
